lib/http/endpoint.js

```diff
@@ -200,7 +200,7 @@ const defaultResultWriter = (result, request, response, next) => {
     result.with(response).pipeline((err) => next?.(err));
```

Comment:
I would expect the check to also be required here, but adding it doesn't make any different to errors logged by the integration test suite.